bot/exts/evergreen/avatar_modification/_effects.py

```diff
@@ -96,6 +96,17 @@ def eight_bitify_effect(image: Image.Image) -> Image.Image:
         image = image.resize((1024, 1024), resample=Image.NEAREST)
         return image.quantize()
 
+    @staticmethod
+    def flip_effect(image: Image.Image) -> Image.Image:
+        """
+        Flips the image horizontally.
+
+        This is done by just using ImageOps.flip().
```

Comment:
This should be mirror

```suggestion
        This is done by just using ImageOps.mirror().
```